### DISCUSSION TITLE
How to see progress and which parameters to use.

### DISCUSSION THREAD

**Discussion by Ma1Dong** [projectdiscovery/nuclei-templates#6398] [Q&A] [answered]
How to see progress and which parameters to use。For example, I want to know what percent of the scan is

Just like this
![image](https://user-images.githubusercontent.com/52943794/208666985-3c6d88b1-c118-49a0-a118-4f77552cdcec.png)

**Comment by olearycrew** [comment, marked as answer]
For this, you can use the `-stats` flag when running Nuclei. Hope that helps!
